.github/workflows/wheels.yml

```diff
@@ -5,9 +5,70 @@ on:
     tags:
       - '*'
 jobs:
+  build_wheels:
+    name: Build wheels
+    runs-on: ${{ matrix.os }}
+    environment: release
+    permissions:
+      id-token: write
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.10'
+      - uses: dtolnay/rust-toolchain@stable
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.13.0
+      - uses: actions/upload-artifact@v3
+        with:
+          path: ./wheelhouse/*.whl
+      - name: Publish package distributions to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          packages-dir: wheelhouse/
+  build_wheels_macos_arm:
+    name: Build wheels on macOS arm
+    runs-on: ${{ matrix.os }}
+    environment: release
+    permissions:
+      id-token: write
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Looks like the OS doesn't match the job here.

Reply:
Nice catch! It was a sloppy copy and paste on my part.

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/10999/commits/6963092f351540308d3b1c3bf7bd0464f698c5c1